pkg/ccr/record/alter_view.go

```diff
@@ -0,0 +1,31 @@
+package record
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type AlterView struct {
+	DbId          int64    `json:"dbId"`
+	TableId       int64    `json:"tableId"`
+	InlineViewDef string   `json:"inlineViewDef"`
+	SqlMode       int64    `json:"sqlMode"`
+}
+
+func NewAlterViewFromJson(data string) (*AlterView, error) {
+	var alterView AlterView
+	err := json.Unmarshal([]byte(data), &alterView)
+	if err != nil {
+		return nil, fmt.Errorf("unmarshal alter view error: %v", err)
+	}
+
+	if alterView.TableId == 0 {
+    	return nil, fmt.Errorf("table id not found")
```

Comment:
format it

Reply:
Thank you for your comment. I have made the necessary changes